applications/PoromechanicsApplication/custom_problemtype/Poromechanics_Application.gid/Poromechanics_Application.prb

```diff
@@ -103,9 +103,9 @@ QUESTION: Move_Mesh#CB#(true,false)
 VALUE: false
 QUESTION: Block_Builder#CB#(true,false)
 VALUE: true
-QUESTION: Solver_Type#CB#(SkylineLUFactorizationSolver,SuperLUSolver,BICGSTABSolver,AMGCL,Klu,AztecSolver,AmgclMPISolver,MultiLevelSolver)
-VALUE: AMGCL
-HELP: OpenMP solvers: SkylineLUFactorizationSolver (Direct), SuperLUSolver (Direct), BICGSTABSolver (Iter.) and AMGCL (Iter.). MPI solvers: Klu,AztecSolver,AmgclMPISolver,MultiLevelSolver.
+QUESTION: Solver_Type#CB#(skyline_lu_factorization,super_lu,bicgstab,amgcl,Klu,AztecSolver,AmgclMPISolver,MultiLevelSolver)
+VALUE: amgcl
+HELP: OpenMP solvers: skyline_lu_factorization (Direct), super_lu (Direct), bicgstab (Iter.) and amgcl (Iter.). MPI solvers: Klu,AztecSolver,AmgclMPISolver,MultiLevelSolver.
```

Comment:
the names for the trilinos-solvers are also updated, see [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/trilinos_application/custom_factories/trilinos_linear_solver_factory.cpp#L40-L78)

Reply:
Ok, I will also update the MPI solvers names then